### PR TITLE
Dark mode fixes to the home page and profile page

### PR DESCRIPTION
**Pull request by ChinmayMhatre**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -10,20 +10,21 @@ module.exports = {
...
+          "low-medium":"rgb(209 213 219)",
+          "medium": "#233042",
+          "high": "rgb(17 24 39)",
         },
         secondary: {
           // purple
-          low: "rgb(165 180 252)",
-          medium: "rgb(79 70 229)",
-          high: "rgb(67 56 202)",
+          "low": "rgb(165 180 252)",
```
Comment (eddiejaoude) [inline tailwind.config.js:20]: `"` are not needed on the property 

```suggestion
          low: "rgb(165 180 252)",
```

**Comment by ChinmayMhatre** [comment]
Wanted opinion on which looks the best


image

image

image

**Comment by ChinmayMhatre** [comment]
Will have to check which colours don't follow the accessibility rule in the tests

**Comment by eddiejaoude** [comment]
I think the 2nd one stands out the most

**Comment by CBID2** [comment]
I vote for number 2 as well @ChinmayMhatre! 😊

**Review by eddiejaoude — APPROVED** [review]
Looks great 👍

**Pull request merged**